### ISSUE TITLE
Fix use of deprecated `hmset` command when running the data loader.

### ISSUE THREAD

**Issue by simonprickett** [redis-developer/introducing-the-geosearch-command#19]
The data loader for this project uses the deprecated `hmset` command.  Fix this to use `hset` appropriately and remove deprecated `hmset`:

```
python load_stations.py
/Users/simonprickett/source/github/introducing-the-geosearch-command/load_stations.py:39: DeprecationWarning: Pipeline.hmset() is deprecated. Use Pipeline.hset() instead.
  pipeline.hmset(station_key, station_data)
Loaded 52 stations.
```

Use the dict mapping with `hset`: https://redis-py.readthedocs.io/en/stable/commands.html?highlight=hset#redis.commands.core.CoreCommands.hset

This is part of our participation in Hacktoberfest 2022. Learn how to complete this issue here: https://redis.io/community/hacktoberfest/

Need help or want to talk to us? Join us on Discord where we have a dedicated Hacktoberfest channel: https://discord.gg/ueQwKUh5Z3

If you're interested in taking this issue on, please mention @simonprickett and @SuzeShardlow in the comments and ask to be assigned.

**Comment by SuzeShardlow** [comment]
This issue is available and up for grabs.  If you would like to work on it, please reply and tag me and @simonprickett :)

**Comment by meirdev** [comment]
Can you assign it to me? @SuzeShardlow @simonprickett

**Comment by SuzeShardlow** [comment]
Hi @meirdev, I've assigned this to you, looking forward to seeing your PR in the next couple of days. Have fun and please check out our guidelines:

https://redis.io/community/hacktoberfest/

**Comment by meirdev** [comment]
#23

**Comment by SuzeShardlow** [comment]
Closed by #23.
